src/esp/gfx/DrawableGroup.h

```diff
@@ -3,22 +3,57 @@
 // LICENSE file in the root directory of this source tree.
 
 #pragma once
-
+#include <Magnum/SceneGraph/Drawable.h>
 #include <Magnum/SceneGraph/FeatureGroup.h>
+#include <Magnum/SceneGraph/SceneGraph.h>
+#include <functional>
```

Comment:
unneeded? this include is [one of the heavier ones](https://artificial-mind.net/projects/compile-health/), so the less is used the better :)